### DISCUSSION TITLE
Be able to start comment then pick account

### DISCUSSION THREAD

**Discussion by ghost** [Docile-Alligator/Infinity-For-Reddit#978] [Ideas]
I like to browse reddit anonymously, but when I want to comment, I have a few accounts to use depending on the subreddit. So what would be convinient is that if you press the comment button while browsing anonymously, there could be a popup to choose an account you are already signed in to, or have a similar one when you attempt to post the comment, or even have one similar to the one when you are logged in, where there is a button to choose an account.

**Comment by Wladefant** [comment]
Part of #932

**Reply by Wladefant** [reply]
@user1027 thank you, you are right
